freqtrade/freqai/prediction_models/LightGBMRegressorMultiTarget.py

```diff
@@ -31,6 +32,9 @@ def fit(self, data_dictionary: Dict) -> Any:
         eval_set = (data_dictionary["test_features"], data_dictionary["test_labels"])
         sample_weight = data_dictionary["train_weights"]
 
+        if self.continual_learning:
+            logger.warning('Continual learning not supported for MultiTarget models')
```

Comment:
wouldn't you kindof want to warn about this on startup?
I think with this it's going to be shown once per pair per training ...

Reply:
I just went ahead and made our own FreqAI regressor to allow us to support continual learning and eval sets on multioutput :-)